examples/gallery/build.rs

```diff
@@ -2,5 +2,10 @@
 // SPDX-License-Identifier: MIT
 
 fn main() {
-    slint_build::compile("gallery.slint").unwrap();
+    slint_build::compile_with_config(
+        "gallery.slint",
+        slint_build::CompilerConfiguration::new()
+            .with_bundled_translations(concat!(env!("CARGO_MANIFEST_DIR"), "/lang/")),
```

Comment:
the printerdemo only does it for wasm and android.  https://github.com/slint-ui/slint/blob/55a45a6c339223a19bcd79fc191e79193f495a79/demos/printerdemo/rust/build.rs#L7

I think we should do the same.